File: java/TrunkMaps.java
public class TrunkMaps {
    public static void main(String[] args) throws Exception {
        String ex = System.getenv("EXOPHONE");
        if (ex == null || ex.isEmpty()) ex = System.getenv("DID_NUMBER");
        if (ex == null || ex.isEmpty()) {
            System.err.println("Error: set EXOPHONE or DID_NUMBER");
            System.exit(1);
        }
        StringBuilder q = new StringBuilder("exophone=").append(java.net.URLEncoder.encode(ex, "UTF-8"));
        String ts = System.getenv("TRUNK_SID");
        if (ts != null && !ts.isEmpty()) q.append("&trunk_sid=").append(java.net.URLEncoder.encode(ts, "UTF-8"));
        System.out.println("Trunk map lookup...");
        _Client.get("/trunk-maps", q.toString());
    }
}

File: php/trunk_maps.php
#!/usr/bin/env php
<?php
require '_client.php';
$ex = getenv('EXOPHONE') ?: getenv('DID_NUMBER');
if (!$ex) {
    echo "Error: set EXOPHONE or DID_NUMBER\n";
    exit(1);
}
$q = ['exophone' => $ex];
if (getenv('TRUNK_SID')) {
    $q['trunk_sid'] = getenv('TRUNK_SID');
}
echo "Trunk map lookup...\n";
exo_get('/trunk-maps', $q);
